scripts/confirmJenkinsConfigurations.sh

```diff
@@ -0,0 +1,43 @@
+set -e
+
+if [ "$#" -ne 4 ]; then
+    echo "This script is used during the SetupReleaseConfig step in Jenkins. It confirms correct configurations that will be used by Jenkins during Reactome's data release."
+    echo ""
+    echo "Usage: 'bash confirmJenkinsConfigurations.sh configFilepath jenkinsEnvironmentReleaseNumber jenkinsEnvironmentPreviousReleaseNumber userInputReleaseNumber'"
+    exit 1
+fi
+
+configFile=$1
+environmentalReleaseNumber=$2
+environmentalPrevReleaseNumber=$3
+userInputReleaseNumber=$4
+
+configFileReleaseNumber=
+
+while read line; do
+  if [[ $line =~ releaseNumber= ]] ; then
+    configFileReleaseNumber=${line#*=}
+  fi
+done < $configFile
```

Comment:
Could you add some comments to detail what the loop is iterating over and what is being done inside to decide what value `configFileReleaseNumber` receives?